pkgs/by-name/lo/lofreq/package.nix

```diff
@@ -0,0 +1,60 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+
+  autoreconfHook,
+  htslib,
+  python3,
+  zlib,
+
+  nix-update-script,
+}:
+
+stdenv.mkDerivation rec {
```

Comment:
Lets use `finalAttrs` instead of `rec`
- https://github.com/NixOS/nixpkgs/issues/315337